test/mocks/network_manager_mock.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:flutterapp/data/network_manager.dart';
+import 'package:mockito/mockito.dart';
+
+NetworkManager getConnectionManager(bool hasConnection) {
```

Comment:
Are there any libraries that allow us to create mocks as Mockito does?

Reply:
You can use Mockito on Flutter as well.